python/tests/test_table.py

```diff
@@ -90,7 +90,20 @@ def test_add(db):
             {"vector": [5.9, 26.5], "item": "bar", "price": 20.0},
         ],
     )
+    _add(table)
 
+    # from empty table GH#54
+    table = LanceTable.create(db, "test2")
```

Comment:
in this moment, what would be the behavior to call `table.schema()` `table.count_rows()` and etc?

Reply:
That behavior is already allowed by having the data and schema be optional to the creation of the table.